src/utils/wallet.js

```diff
@@ -6,53 +6,51 @@
  */
 
 import {
-  SENTRY_DSN,
   DEBUG_LOCAL_DATA_KEYS,
-  WALLET_HISTORY_COUNT,
   METADATA_CONCURRENT_DOWNLOAD,
+  SENTRY_DSN,
+  WALLET_HISTORY_COUNT,
   WALLET_SERVICE_MAINNET_BASE_URL,
   WALLET_SERVICE_MAINNET_BASE_WS_URL,
 } from '../constants';
 import { FeatureFlags } from '../featureFlags';
 import STORE from '../storageInstance';
 import store from '../store/index';
 import {
-  setWallet,
-  updateLoadedData,
+  cleanData,
+  historyUpdate,
   isOnlineUpdate,
-  updateHeight,
-  updateTx,
   loadingAddresses,
   loadWalletSuccess,
-  historyUpdate,
-  sharedAddressUpdate,
-  reloadData,
-  cleanData,
-  changeServer,
-  updateTokenHistory,
-  tokenMetadataUpdated,
+  lockWalletForResult,
   metadataLoaded,
   partiallyUpdateHistoryAndBalance,
+  reloadData,
   setUseWalletService,
-  lockWalletForResult,
+  setWallet,
+  sharedAddressUpdate,
+  tokenMetadataUpdated,
+  updateHeight,
+  updateLoadedData,
+  updateTokenHistory,
+  updateTx,
 } from '../actions/index';
 import {
-  helpers,
+  config,
+  Connection,
   constants as hathorConstants,
   errors as hathorErrors,
   HathorWallet,
   HathorWalletServiceWallet,
-  Connection,
+  helpers,
+  metadataApi,
   Network,
-  wallet as oldWalletUtil,
-  walletUtils,
   storage,
   tokens,
-  metadataApi,
-  config,
+  wallet as oldWalletUtil,
+  walletUtils,
 } from '@hathor/wallet-lib';
```

Comment:
_Note:_ the IDE linter reordered alphabetically the object destructuring made on the imports here. That's why all these lines were changed.